homeassistant/components/tuya/select.py

```diff
@@ -146,6 +147,13 @@
             device_class=DEVICE_CLASS_TUYA_MOTION_SENSITIVITY,
             entity_category=EntityCategory.CONFIG,
         ),
+        SelectEntityDescription(
+            key=DPCode.PIR_SWITCH,
+            name="PIR Detection Switch",
+            icon="mdi:motion-sensor",
+            device_class=DEVICE_CLASS_TUYA_PIR_SWITCH,
```

Comment:
We can re-use the existing device class here:

```suggestion
            device_class=DEVICE_CLASS_TUYA_MOTION_SENSITIVITY,
```